src/lib/realtime.js

```diff
@@ -0,0 +1,281 @@
+/* global WebSocket */
```

Comment:
I think we may remove this file, summon the wrath of `git commit --fixup` !

Reply:
Indeed :D